tools.py

```diff
@@ -46,12 +46,25 @@ def cut(input_path, output_path, sample_rate, mono, dilate, strip, strip_prefix,
 		segment_path = os.path.join(output_path, os.path.basename(audio_path) + '.{channel}-{begin:.06f}-{end:.06f}.wav'.format(**t))
 		audio.write_audio(segment_path, segment, sample_rate, mono = True)
 		if strip_prefix:
-			segment_path = segment_path.replace('data/', '')
-		t = dict(audio_path = segment_path, channel = 0 if len(signal) == 1 else None, begin = 0.0, end = segment.shape[-1] / sample_rate, speaker = t.pop('speaker', None), ref = t.pop('ref'), hyp = t.pop('hyp', None), cer = t.pop('cer', None), alignment = t.pop('alignment', {}), words = t.pop('words', {}), meta = t)
+			segment_path = segment_path.lstrip(strip_prefix)
```

Comment:
`lstrip` will not work reliably to remove the prefix, it removes the passed character set without considering the ordering